UnitTests/BraintreeCoreTests/BTAppContextSwitch_Tests.swift

```diff
@@ -0,0 +1,111 @@
+import XCTest
+
+class BTAppContextSwitch_Tests: XCTestCase {
```

Comment:
The diff shows this as a new file, but we actually just renamed `BTAppSwitch_Tests.swift`.